nautobot/docs/additional-features/jobs.md

```diff
@@ -451,19 +452,21 @@ class NewBranch(Job):
 A job to perform various validation of Device data in Nautobot. As this job does not require any user input, it does not define any variables, nor does it implement a `run()` method.
 
 ```python
-from nautobot.dcim.choices import DeviceStatusChoices
 from nautobot.dcim.models import ConsolePort, Device, PowerPort
+from nautobot.extras.models import Status
 from nautobot.extras.jobs import Job
 
 
+STATUS_ACTIVE = Status.objects.get(slug='active')
```

Comment:
I think in general we want to avoid having jobs call into the ORM as a side effect of simply being loaded. This belongs in either a `test_*` method or a `run` method.